src/e2e.js

```diff
@@ -36,12 +36,21 @@ mye2e.prototype.setup = function(passphrase, userid) {
   return Promise.resolve();
 };
 
+mye2e.prototype.clear = function() {
+  // e2e can only store one private key in LocalStorage
+  // Attempting to set another will result in an HMAC error
+  // So, make sure to clear before doing so
+  // See googstorage.js for details on how storage works
+  store.prototype.clear();
+};
+
 mye2e.prototype.importKeypair = function(passphrase, userid, privateKey) {
+  this.clear();
```

Comment:
Import keypair sets a new private key, and since e2e can only support a single private key on the keyring it needs to be cleared first. The e2e issue says that this is a purposeful design decision, and suggests using multiple browser profiles if multiple keyrings are desired.